public/data/damoski2.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "damoski2",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/damoski2.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/damoski2",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AkogunOyindamo3",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/oyindacodes/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Check Out my Portfolio",
+      "url": "https://aqua-portfolio.vercel.app/",
+      "icon": "web"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "May 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started freelancing",
+      "url": "https://www.linkedin.com/feed/"
```

Comment:
Are you sure about this? I think this might be a mistake

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Brittle

Reply:
Good catch Krish!

Reply:
It looks like this is their LinkedIn profile: https://www.linkedin.com/in/oyindacodes